### ISSUE TITLE
Take a look to my "improvements"

### ISSUE THREAD

**Issue by ghost** [ge2301/Nereus_Kodi#2]
https://github.com/xstreamon/Nereus_Kodi/tree/PVR

Just a mix of yours and pkscout's. But I think it is a good way to proceed....

**Comment by ge2301** [comment]
Do you have an user in Kodi forum to communicate?

**Comment by ghost** [comment]
Nope . But you can communicate Grabber66. He is at Kodinerds and Kodi.tv....
We are using one PC.....

After all i have a problem with this version. It works fine but if i install the zipped file to another installation kodi breaks ????
Aktually i try out what's the Problem is....

**Comment by ge2301** [comment]
Yeah, there are several issus I wanted to report with weather.metoffice "ERROR: CSettingString: unknown options filler " and wrongly named fonts "ERROR: Couldn't load font name: big_time(noto-sans-bold.ttf), trying to substitute arial.ttf" generally the fonts are names w/o the first '-", means e.g notosans-bold.ttf. Other points I could not check, because the skin makes directly Kodi crash.
